scalafix-sbt/src/main/scala/scalafix/internal/sbt/ScalafixRuleNames.scala

```diff
@@ -2,27 +2,28 @@
 package scalafix.internal.sbt
 
 object ScalafixRuleNames {
-  def all: List[String] = List(
-    "DottyKeywords",
-    "DisableSyntax",
-    "NoFinalize",
-    "NoValInForComprehension",
-    "RemoveXmlLiterals",
-    "LeakingImplicitClassVal",
-    "VolatileLazyVal",
-    "ProcedureSyntax",
-    "ExplicitUnit",
-    "DottyVolatileLazyVal",
-    "DottyVarArgPattern",
-    "NoInfer",
-    "Sbt1",
-    "ExplicitResultTypes",
-    "ExplicitReturnTypes",
-    "RemoveUnusedImports",
-    "RemoveUnusedTerms",
-    "NoAutoTupling",
-    "Disable",
-    "DisableUnless",
-    "MissingFinal"
+  def all: List[(String, String)] = List(
+    ("DottyKeywords", "Rewrite that replaces enum and inline with `enum` and `inline` for compatibility with Dotty"),
```

Comment:
Hum, this should be available in the Rule's Description. Is this to avoid classloading the rule?

Reply:
This file is code generated with a task in build.sbt, it's to avoid classloading yes. Otherwise we'd need a `Task[Parser]` and I had no idea how to make that work

Reply:
I think it's worth investigating in a separate issue.